src/main/webapp/app/overview/course-conversations/course-conversations.module.ts

```diff
@@ -33,6 +33,7 @@ import { ProfilePictureComponent } from 'app/shared/profile-picture/profile-pict
 import { SavedPostsComponent } from 'app/overview/course-conversations/saved-posts/saved-posts.component';
 import { PostingSummaryComponent } from 'app/overview/course-conversations/posting-summary/posting-summary.component';
 import { InfiniteScrollDirective } from 'ngx-infinite-scroll';
+import { ForwardMessageDialogComponent } from 'app/overview/course-conversations/dialogs/forward-message-dialog/forward-message-dialog.component';
```

Comment:
We actually want to get rid of Angular client modules. Would it make sense to remove this one here or in another PR?

Reply:
I removed `ForwardMessageDialogComponent` from this module. I think additional changes can be made in follow-up PRs.